File: eric-emery/tip-calculator.js
function calculateTip(amount, rating) {

var rate = rating.toLowerCase();

if(rate === "excellent"){
  return Math.ceil(amount*20/100);
}
else if(rate === "great"){
  return Math.ceil(amount*15/100);
}

else if(rate === "good"){
  return Math.ceil(amount*10/100);
}

else if(rate === "poor"){
  return Math.ceil(amount*5/100);
}

else if(rate === "terrible"){
  return Math.ceil(amount*0/100);
}

else{
return("Rating not recognised");
}
  
}

/* PSUEDOCODE

Project needs to define whether not the tip is good or bad
-All decisions are based of of percentages
-Low percentage tips are terrible-poor
-Medium-High percentage tips are good-excellent

  Coding :(
  
  -We need to define functions that will print different strings
  depending on the tip percentage.
  -POSSIBLY USE IF ELSE STATEMENTS?

File: parker-senter/tip-calculator.py
#pulls math in
import math
def calculate_tip(amount, rating):
    # checks what the rating is and gives the right tip amount
    if rating.lower() == "terrible":
        return 0
    #sets the rating to all lowercase
    elif rating.lower() == "poor":
        #rounds up to the nearest whole number
        return math.ceil(amount * 0.05)
    elif rating.lower() == "good":
        return math.ceil(amount * 0.1)
    elif rating.lower() == "great":
        return math.ceil(amount * 0.15)
    elif rating.lower() == "excellent":
        return math.ceil(amount * 0.2)
    else:
        return "Rating not recognised"
    
"""
Pusedocode
takes in the rating 
turns the rating to lower case
checks what the rating is
takes the total and multiplies it by the tip amount
rounds the tip up to the nearest whole number
"""
